src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -9,6 +9,17 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
   text-align: center;
 }
 
+.FileUploadSmallView {
+  padding: 1.25rem;
+}
+
+.Content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
```

Comment:
I might be the tiniest bit worried about this but not sure how to test it. Going to do a 🎩  and play with some of the values